nrf_modem/doc/gnss_interface.rst

```diff
@@ -0,0 +1,534 @@
+
+.. note::
+
+   QZSS is only supported by modem firmware v1.3.0 or later.
+
+PRN mask
+--------
+
+QZSS satellite acquisition and tracking can be configured per satellite using QZSS PRN mask. Bits 0...9 correspond to QZSS PRNs
+193...202, respectively. When a bit is set, usage of the corresponding QZSS satellite is enabled. Bits 10...15 are reserved and
+their value is ignored.
+
+The default is that all QZSS PRNs (193...202) are enabled.
+
+QZSS PRNs 193, 194, 195 and 199 can be enabled (and others disabled) as shown in the following example:
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_qzss_prn_mask_set(0x47);
+
+.. note::
+
+   QZSS is only supported by modem firmware v1.3.0 or later.
+
+Enabling GNSS priority mode
+***************************
+
+GNSS can be given priority over LTE idle mode procedures to help getting a fix. Usually this is not necessary when eDRX and/or
+PSM is used, but if that's not possible, the GNSS priority mode may be used.
+
+Priority for GNSS should be used only when fix has been blocked by LTE idle mode operations, which can be detected by
+NRF_MODEM_GNSS_PVT_FLAG_NOT_ENOUGH_WINDOW_TIME bit being set in the PVT data frame flags member. The application should not
+make the decision based on a single PVT event, but should enable priority only in case this flag has been set in several
+consecutive PVT events.
+
+Priority mode is disabled automatically after producing the first fix. It can also be disabled by the application by calling
+:c:func:`nrf_modem_gnss_prio_mode_disable` function.
+
+.. note::
+
+   GNSS priority may interfere with LTE operations, if possible it would be good to time the use of priority to moments where
+   data transfer is not anticipated. In general, long enough eDRX cycles, or PSM, will ensure better functionality for both
+   GNSS and LTE.
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_prio_mode_enable();
+
+Changing the dynamics mode
+**************************
+
+Dynamics mode can be used to tune GNSS performance for a specific use case. Use of a matching dynamics mode improves the
+positioning performance. Dynamics mode can be changed without disruption in positioning. The selected dynamics mode is stored
+into non-volatile memory.
+
+The default value is NRF_MODEM_GNSS_DYNAMICS_GENERAL_PURPOSE.
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_dyn_mode_change(NRF_MODEM_GNSS_DYNAMICS_AUTOMOTIVE);
+
+.. note::
+
+   Dynamics mode is only supported by modem firmware v1.3.0 or later.
+
+1PPS
+****
+
+GNSS can provide time synchronized eletrical pulses to the COEX1 pin. The rising edge of the pulse is aligned as closely as
+possible to the GPS time second.
+
+The pulse interval and width are configurable. It's also possible to configure the pulses to start at a specific date and time.
+Instead of repeating pulses 1PPS can also be used in a one-time pulse mode, where only a single pulse is given at the specified
+time or as soon as GNSS gets a fix.
```

Comment:
```suggestion
The pulse interval and width are configurable.
It is also possible to configure the pulses to start at a specific date and time.
Instead of repeating pulses, 1PPS can also be used in a one-time pulse mode, where only a single pulse is given at the specified time or as soon as GNSS gets a fix.
```